utils/nx-plugin/index.js

```diff
@@ -0,0 +1,13 @@
+const {
+  addReleaseBuildTargets,
+} = require('./clone-build-targets-for-release.cjs');
+
+module.exports = /** @type {import('@nrwl/devkit').NxPlugin} */ ({
+  name: '@coveo/nx-plugin',
```

Comment:
can we have a more specific name e.g. @coveo/ui-kit-release-bump

Reply:
I'm a bit hesitant about this. If we need to change other things with Nx, I'm tempted to re-use the same plugin to avoid polluting the repo with too many projects and having to copy and paste configs.

On the other hand, this currently only has one function and we ideally want to avoid needing nx plugins. If I rename this project, I would also move all the code from `clone-build-targets-for-release.cjs` into this file.